File: level 0/두 수의 나눗셈.py
def solution(num1, num2):
    answer = int(str(num1 / num2 * 1000).split(".")[0])
    return answer

# 더 좋은 풀이
# def solution(num1, num2):
#     return int(num1 / num2 * 1000)     그냥 int()만 써주면 자동으로 소숫점 아래 부분은 사라지고 정수 부분만 남음.

File: javascript_programmers_school/level 0/두 수의 나눗셈.js
const solution = (num1, num2) => Math.floor(num1 / num2 * 1000)

// Math.floor 메서드
// Math.floor(3.5) = 3
// Math.floor(3.87) = 3
// Math.floor(-3.5) = -4     --> 더 작은 정수 중 입력값과 가장 가까운 정수
// Math.floor(-3.14) = -4

// Math.turnc 메서드
// Math.turnc(3.5) = 3
// Math.turnc(3.87) = 3
// Math.turnc(-3.5) = -3     --> 그냥 소수점 아래만 삭제
// Math.turnc(-3.14) = -3